Spatial.gd

```diff
@@ -17,8 +17,7 @@ var next_texture_path: String = ""
 # A pointer to the path object that any new nodes should be appended to. If
 # the value is null then a new path will be created the next time a path point
 # will be created.
-var currently_active_path = null
-var currently_active_path2d = null
+var currently_active_path_index = null
 var currently_active_category = null
```

Comment:
If currently_active_category is changed or unset does anything else need to be changed or unset with it?